resource/specs/testnet.toml

```diff
@@ -50,27 +50,27 @@ secp256k1_ripemd160_sha256_sighash_all = [
 ]
 
 [genesis.bootstrap_lock]
-code_hash = "0x54811ce986d5c3e57eaafab22cdd080e32209e39590e204a99b32935f835a13c"
+code_hash = "0x68d5438ac952d2f584abf879527946a537e82c7f3c1cbf6d8ebf9767437d8e88"
 # Lock for developers to run tests
 args = ["0x64257f00b6b63e987609fa9be2d0c86d351020fb"]
 hash_type = "data"
 
 # Locks for developers to run tests
 [[genesis.issued_cells]]
 capacity = 50_000_000_00000000
-lock.code_hash = "0x54811ce986d5c3e57eaafab22cdd080e32209e39590e204a99b32935f835a13c"
+lock.code_hash = "0x68d5438ac952d2f584abf879527946a537e82c7f3c1cbf6d8ebf9767437d8e88"
 lock.args = ["0x64257f00b6b63e987609fa9be2d0c86d351020fb"]
-lock.hash_type = "data"
+lock.hash_type = "type"
```

Comment:
Why `code_hash` is the same but `hash_type` is different?

Reply:
Updated, please review agian, thanks.